runtime/src/accounts_db.rs

```diff
@@ -4246,10 +4246,9 @@ impl AccountsDb {
     }
 
     fn get_storages_for_slot(&self, slot: Slot) -> Option<SnapshotStorage> {
-        self.storage.map.get(&slot).map(|storages| {
+        self.storage.get(&slot).map(|storages| {
             // per slot, get the storages. There should usually only be 1.
             storages
-                .value()
```

Comment:
compiles with or without this `value()`. `value()` returns `&V` from the entry. I assume that otherwise, the methods on `&V` are accessible to the `Ref` we are returning. This is similar to how you can call methods on `X` if your type is `Arc<X>`